bug: WebSearch tool timeout causes run failure

## Summary
The `WebSearch` tool exceeded the 60-second timeout limit during a run, causing the sandbox to kill the process and mark the run as `failed`.

## Affected Run
- **Run ID:** `12afaff5-2f5f-4006-ab73-2d0a7cbfd844`
- **Agent:** Connector Master
- **User:** `ethan@vm0.ai` (vm0 main org)
- **Prompt:** \"帮我看看 box.net 能不能做成一个 api token connector\"
- **Time:** 2026-04-21 09:21 UTC

## Error Details
```
[2026-04-21T09:23:30.119Z] [WARN] Tool timeout: WebSearch stuck for 64s, killing process
[2026-04-21T09:23:30.136Z] [ERROR] CLI execution failed: execution: Tool timeout: WebSearch exceeded 60s without returning a result
```

## Impact
- Run duration: ~2 min 17 sec before failure
- Error Report reference: `er-f4f3c468` (auto-generated and sent to Plain support)
- This appears to be a transient network/dependency issue where the WebSearch provider failed to respond within the timeout window.

## Possible Actions
1. Investigate WebSearch provider latency and reliability
2. Consider adding retry logic with exponential backoff for WebSearch
3. Evaluate whether the 60s timeout is appropriate for this tool

--- a/crates/guest-agent/src/constants.rs
+++ b/crates/guest-agent/src/constants.rs
@@ -29,7 +29,7 @@ pub const HTTP_UPLOAD_TIMEOUT_SECS: u64 = 60;
 /// Kill the CLI process if a network tool hasn't returned a result within
 /// this duration.  Override with `VM0_STUCK_TOOL_TIMEOUT_SECS` env var.
 /// See: https://github.com/anthropics/claude-code/issues/11650
-pub const STUCK_TOOL_TIMEOUT_SECS: u64 = 60;
+pub const STUCK_TOOL_TIMEOUT_SECS: u64 = 180;
 
 /// How often (in seconds) to check for stuck tools in the select loop.
 pub const STUCK_TOOL_CHECK_INTERVAL_SECS: u64 = 5;

